SynonymCreator/src/main/scala/no/nrk/samnorsk/synonymmapper/ApertiumRunner.scala

```diff
@@ -17,28 +17,29 @@ abstract class ApertiumRunner {
   }
 }
 
-class LocalApertiumRunner(fromLanguage: String, toLanguage: String) extends ApertiumRunner {
+class LocalApertiumRunner(fromLanguage: Language, toLanguage: Language) extends ApertiumRunner(fromLanguage, toLanguage) {
   override def translate(text: String): String = {
-    val tempFile = File.createTempFile("apertium-input", fromLanguage)
+    val tempFile = File.createTempFile("apertium-input", fromLanguage.Name)
 
     try {
       Files.write(tempFile.toPath, text.getBytes(StandardCharsets.UTF_8))
-      s"apertium $fromLanguage-$toLanguage ${tempFile.getAbsolutePath}".!!.trim
+      s"apertium ${fromLanguage.Apertium}-${toLanguage.Apertium} ${tempFile.getAbsolutePath}".!!.trim
     } finally {
       Files.delete(tempFile.toPath)
     }
   }
 }
 
 // not tested, may not work
-class RemoteApertiumRunner(fromLanguage: String, toLanguage: String, user: String, server: String, keyFile: Path)
-  extends ApertiumRunner {
+class RemoteApertiumRunner(fromLanguage: Language, toLanguage: Language, user: String, server: String, keyFile: Path)
+  extends ApertiumRunner(fromLanguage, toLanguage) {
   override def translate(text: String): String = {
-    s"""ssh $user@$server "echo \\"$text\\" | apertium $fromLanguage-$toLanguage"""".!!.trim
+    s"""ssh $user@$server "echo \\"$text\\" | apertium ${fromLanguage.Apertium}-${toLanguage.Apertium}"""".!!.trim
```

Comment:
""" means multiline comment, if its the var subst you need its the s infront, so use s"whatever $var you want" to increase readability